craft_application/fetch.py

```diff
@@ -294,6 +295,11 @@ def configure_instance(
     return net_info.env
 
 
+def get_log_filepath() -> pathlib.Path:
+    """Get the path containing the fetch-service's output."""
+    return pathlib.Path(tempfile.gettempdir()) / "craft-fetch-log.txt"
```

Comment:
Will it work with `tempfile` ?
I though that snaps have `tempfiles` isolated from each other.
Do we have any integration test, that checks if log are collected ?

Reply:
Hmm could you run it using `snap run` instead of using it's name ?
I'm a bit concert that it works, cause it would suggest that somehow confinement is compromised :sweat_smile: